silverback/middlewares.py

```diff
@@ -119,4 +113,9 @@ async def on_error(
         result: TaskiqResult,
         exception: BaseException,
     ):
-        logger.error(f"{message.task_name} - {type(exception).__name__}: {exception}")
+        percentage_time = 100 * (result.execution_time / self.block_time)
```

Comment:
**potential issue**: In dev / test networks, block time is `0`, which would cause a `DivisionByZero` error here

Reply:
Resolved in [9ca7692](https://github.com/ApeWorX/silverback/pull/66/commits/9ca769275d2a05196c23975807b1f112e4549e3b), also led to me finding another bug